src/src/_lib/index.ts

```diff
@@ -0,0 +1,69 @@
+import * as Luxon from 'luxon'
+
+import {CalendarSelectionRangeI} from '../types'
+
+export function getCurrentTime(): Luxon.DateTime {
+  return Luxon.DateTime.local()
+}
+
+export function getDifferenceInDays({start, end}: CalendarSelectionRangeI): number {
+  const difference = end.diff(start).as('days')
+
+  return difference > 0 ? difference : 0
+}
+
+export function getMaxWeekNumberInYear(date: Luxon.DateTime): Number {
+  return date.weeksInWeekYear
+}
+
+export function getEachDayOfInterval({start, end}: CalendarSelectionRangeI): Luxon.DateTime[] {
+  return Array.from(
+    new Array(Math.round(getDifferenceInDays({start: start.startOf('day'), end: end.endOf('day')})))
+  ).map((v, i) => start.plus({days: i}))
+}
+
+export function getWeekNumber(date: Luxon.DateTime): Number {
+  return date.weekNumber
+}
+
+export function isAfter(checkDate: Luxon.DateTime, baseDate: Luxon.DateTime): boolean {
+  const difference = baseDate.diff(checkDate).as('seconds')
+
+  return difference < 0
```

Comment:
`baseDate < checkDate` should work, Luxon supports comparison with regular operators